packages/components/src/components/teaser/teaser.ts

```diff
@@ -0,0 +1,163 @@
+    if (this.resizeObserver) {
+      this.resizeObserver.disconnect();
+    }
+  }
+
+  updateOrientation() {
+    if (this.orientation === 'responsive') {
+      this.currentOrientation = this.offsetWidth > 448 ? 'horizontal' : 'vertical';
+      this.requestUpdate();
+    } else {
+      this.currentOrientation = this.orientation;
+    }
+  }
+
+  render() {
+    if (this.variant === 'white border-neutral-300') {
+      this.inset = true;
+    }
+
+    const slots = {
+      default: this.hasSlotController.test('[default]'),
+      media: this.hasSlotController.test('media'),
+      meta: this.hasSlotController.test('meta'),
+      headline: this.hasSlotController.test('headline')
+    };
+
+    return html`
+      <div
+        class=${cx(
+          'flex',
+          {
+            white: 'bg-white',
+            'white border-neutral-300': 'bg-white border border-neutral-300',
+            'neutral-100': 'bg-neutral-100',
+            primary: 'bg-primary text-white',
+            'primary-100': 'bg-primary-100'
+          }[this.variant],
+          (this.orientation === 'vertical' || this.currentOrientation === 'vertical') && 'flex-col',
+          (this.orientation === 'horizontal' || this.currentOrientation === 'horizontal') && 'flex-row gap-8',
+          (this.orientation === 'horizontal' || this.currentOrientation === 'horizontal') && this.inset && 'py-8 px-10'
+        )}
+        part="base"
+      >
+        <div
+          style=${this.orientation === 'horizontal' || this.currentOrientation === 'horizontal'
+            ? `width: var(--distribution-media, 100%);`
+            : ''}
+          class=${cx(
+            !this.inset && (this.orientation === 'vertical' || this.currentOrientation === 'vertical') && 'mb-4',
+            !slots['media'] && 'hidden'
+          )}
+          part="media"
+        >
+          <slot name="media"></slot>
+        </div>
+
+        <div
+          style=${this.orientation === 'horizontal' || this.currentOrientation === 'horizontal'
+            ? `width: var(--distribution-content, 100%); ${
+                this.inset ? 'width: var(--distribution-content, calc(100% - 2rem));' : ''
+              }`
+            : ''}
+          class=${cx(
+            'flex flex-col text-left',
+            (this.orientation === 'horizontal' || this.currentOrientation === 'horizontal') && `flex flex-col`,
+            (this.orientation === 'vertical' || this.currentOrientation === 'vertical') && this.inset && 'm-4'
+          )}
+          id="content"
+        >
+          <div part="meta" class=${cx('gap-2 mb-4', !slots['meta'] && 'hidden')}>
+            <slot name="meta"></slot>
+          </div>
+
+          <div
+            part="headline"
+            class=${cx(
+              'text-lg font-bold mb-1',
+              this.variant === 'primary' ? 'text-white' : 'text-black',
+              !slots['headline'] && 'hidden'
+            )}
+          >
+            <slot name="headline"></slot>
```

Comment:
```suggestion
            <slot name="headline">Always insert one semantically correct heading element here &lt;h&gt </slot>
```

As we said: We should provide a default value in the component, not in the story. :) Then everyone using the component without looking to Storybook sees it.